pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -352,9 +352,9 @@ public Future<Void> sendMessages(final List<? extends Entry> entries, EntryBatch
         return writeAndFlushPromise;
     }
 
-    private void incrementUnackedMessages(int ackedMessages) {
+    private void incrementUnackedMessages(int unackedMessages) {
```

Comment:
Yeah, but we also call this with ackedMessages as a negative counter. Yep - very confusing. Very